2023-05-15 09:57:51,473 - __init__.py:init_device[line:18] - CRITICAL: 初始化网关设备驱动 start...
2023-05-15 09:57:51,485 - device_manager.py:open[line:67] - ERROR: ERROR   : 设备('光纤通信机101') try to open device
2023-05-15 09:57:51,485 - device_manager.py:open[line:67] - ERROR: ERROR   : 设备('电台数传模块206') try to open device
2023-05-15 09:57:51,486 - device_manager.py:open[line:67] - ERROR: ERROR   : 设备('天通通信模块2') try to open device
2023-05-15 09:57:51,486 - device_manager.py:open[line:67] - ERROR: ERROR   : 设备('光纤通信机100') try to open device
2023-05-15 09:57:51,486 - device_manager.py:open[line:67] - ERROR: ERROR   : 设备('电台数传模块205') try to open device
2023-05-15 09:57:51,486 - device_manager.py:open[line:67] - ERROR: ERROR   : 设备('天通通信模块1') try to open device
2023-05-15 09:57:51,497 - bsp_underwater_acoustic_comm.py:open[line:68] - ERROR: ERROR   : 水声通信设备(端口'COM4' 状态False) try to open failed
2023-05-15 09:57:51,497 - bsp_underwater_acoustic_comm.py:open[line:68] - ERROR: ERROR   : 水声通信设备(端口'/dev/ttyUSB0' 状态False) try to open failed
2023-05-15 09:57:51,497 - device_manager.py:open[line:67] - ERROR: ERROR   : 设备('水声通信设备10') try to open device
2023-05-15 09:57:51,497 - device_manager.py:open[line:67] - ERROR: ERROR   : 设备('水声通信设备20') try to open device
2023-05-15 09:57:51,498 - core_scheduling.py:init_device_list_threadhandle[line:70] - CRITICAL: 完成设备列表初始化，存在8个设备
2023-05-15 09:57:51,505 - device_manager.py:close[line:82] - ERROR: ERROR   : 设备('光纤通信机101') try to close device
2023-05-15 09:57:51,505 - device_manager.py:close[line:82] - ERROR: ERROR   : 设备('电台数传模块206') try to close device
2023-05-15 09:57:51,507 - device_manager.py:close[line:82] - ERROR: ERROR   : 设备('水声通信设备20') try to close device
2023-05-15 09:57:51,507 - device_manager.py:close[line:82] - ERROR: ERROR   : 设备('天通通信模块2') try to close device
2023-05-15 09:57:51,507 - device_manager.py:close[line:82] - ERROR: ERROR   : 设备('光纤通信机100') try to close device
2023-05-15 09:57:51,507 - device_manager.py:close[line:82] - ERROR: ERROR   : 设备('电台数传模块205') try to close device
2023-05-15 09:57:51,507 - device_manager.py:close[line:82] - ERROR: ERROR   : 设备('水声通信设备10') try to close device
2023-05-15 09:57:51,507 - device_manager.py:close[line:82] - ERROR: ERROR   : 设备('天通通信模块1') try to close device
2023-05-15 09:57:51,508 - __init__.py:init_device[line:22] - ERROR: 网关设备驱动异常 end...
2023-05-15 09:57:51,511 - connection_workflow.py:start[line:179] - INFO: Pika version 1.3.1 connecting to ('::1', 5672, 0, 0)
2023-05-15 09:57:51,511 - io_services_utils.py:_on_writable[line:345] - INFO: Socket connected: <socket.socket fd=1104, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 6179, 0, 0), raddr=('::1', 5672, 0, 0)>
2023-05-15 09:57:51,512 - connection_workflow.py:_on_transport_establishment_done[line:428] - INFO: Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001404500C880>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001404500C880> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2023-05-15 09:57:51,527 - connection_workflow.py:_report_completion_and_cleanup[line:293] - INFO: AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001404500C880> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2023-05-15 09:57:51,527 - connection_workflow.py:_report_completion_and_cleanup[line:725] - INFO: AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001404500C880> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2023-05-15 09:57:51,527 - blocking_connection.py:_create_connection[line:453] - INFO: Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001404500C880> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2023-05-15 09:57:51,527 - blocking_connection.py:__init__[line:1264] - INFO: Created channel=1
2023-05-15 09:57:51,531 - core_scheduling.py:monitor_web_task_threadhandle[line:237] - WARNING: SUCCESS : Create Pika server.
